num_list.py

```diff
@@ -8,7 +8,8 @@ def print_each_number():
     # ...etc
     # TODO - Write your code below this line.  (Keep the indentation)
     
-
+for i in nums():
```

Comment:
Check your indentation on this block of code as it is "outside" of the `print_each_number` function definition.